src/NuGetPackageVerifier/third-party.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "prefixWhitelist": [
+    "Libuv"
+    "Microsoft.AspNetCore",
+    "Microsoft.CodeAnalysis.Razor",
+    "Microsoft.Data.Sqlite",
+    "Microsoft.EntityFrameworkCore",
+    "Microsoft.Extensions.",
+    "Microsoft.Net.Http.Headers",
+    "Microsoft.NETCore.App",
+    "Microsoft.Owin.Security.Interop",
+    "Microsoft.VisualStudio.Web.",
+    "NETStandard.Library",
+    "System.",
+  ],
+  "prefixBlacklist":  [
+    "System.Ben"
+  ],
+
+  "packages": {
+    "Microsoft.ApplicationInsights.AspNetCore": [ "2.0.0" ],
+    "Microsoft.AspNet.Identity.EntityFramework": [ "2.2.1" ],
+    "Microsoft.AspNet.WebApi.Client": [ "5.2.2" ],
+    "Microsoft.Azure.KeyVault": [ "2.0.6" ],
+    "Microsoft.Build.Runtime": [ "15.2.0-preview-000047-02" ],
+    "Microsoft.CodeAnalysis.Common": [ "1.3.0" ],
+    "Microsoft.CodeAnalysis.CSharp": [ "1.3.0", "2.0.0" ],
+    "Microsoft.CodeAnalysis.CSharp.Workspaces": [ "2.0.0" ],
+    "Microsoft.CodeAnalysis.EditorFeatures.Text": [ "2.0.0-rc5-61501-05" ],
+    "Microsoft.CodeAnalysis.Remote.Razor.ServiceHub": [ "2.0.0-rc5-61501-05" ],
+    "Microsoft.CodeAnalysis.Workspaces.Common": [ "1.3.0", "2.0.0-rc5-61501-05" ],
+    "Microsoft.CSharp": [ "4.3.0" ],
+    "Microsoft.DotNet.InternalAbstractions": [ "1.0.0" ],
+    "Microsoft.IdentityModel.Clients.ActiveDirectory": [ "3.13.5" ],
+    "Microsoft.IdentityModel.Protocols.OpenIdConnect": [ "2.1.3" ],
+    "Microsoft.NETCore.Windows.ApiSets": [ "1.0.1" ],
+    "Microsoft.Owin.Security": [ "3.0.1" ],
+    "Microsoft.Owin.Security.Cookies": [ "3.0.1" ],
+    "Microsoft.VisualStudio.LanguageServices": [ "2.0.0-rc5-61501-05" ],
+    "Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient": [ "2.0.0-rc5-61501-05" ],
+    "Microsoft.VisualStudio.OLE.Interop": [ "7.10.6070" ],
+    "Microsoft.VisualStudio.Shell.14.0": [ "14.3.25407" ],
+    "Microsoft.VisualStudio.Shell.Immutable.10.0": [ "10.0.30319" ],
+    "Microsoft.VisualStudio.Shell.Immutable.11.0": [ "11.0.50727" ],
+    "Microsoft.VisualStudio.Shell.Immutable.12.0": [ "12.0.21003" ],
+    "Microsoft.VisualStudio.Shell.Immutable.14.0": [ "14.3.25407" ],
+    "Microsoft.VisualStudio.Shell.Interop": [ "7.10.6071" ],
+    "Microsoft.VisualStudio.Shell.Interop.10.0": [ "10.0.30319" ],
+    "Microsoft.VisualStudio.Shell.Interop.11.0": [ "11.0.61030" ],
+    "Microsoft.VisualStudio.Shell.Interop.12.0": [ "12.0.30110" ],
+    "Microsoft.VisualStudio.Shell.Interop.8.0": [ "8.0.50727" ],
+    "Microsoft.VisualStudio.Shell.Interop.9.0": [ "9.0.30729" ],
+    "Microsoft.VisualStudio.Text.Data": [ "15.0.26201" ],
+    "Microsoft.Web.Administration": [ "7.0.0" ],
+    "Microsoft.Win32.Registry": [ "4.3.0" ],
+    "Newtonsoft.Json": [ "8.0.3", "10.0.1" ],
```

Comment:
Shouldn't this just allow `10.0.1` and have exclusions in specific repos that needs `8.0.3`?